### DISCUSSION TITLE
Management Group entities are not getting populated for second tenant

### DISCUSSION THREAD

**Discussion by kiranu4u** [Azure/CCOInsights#231] [Q&A] [answered]
I have access to two tenants/directories, when i test the below api getting results from both tenants individually, but when in CCO Dashboard its returning only one tenant management group entities. I tried cleaning the permissions but didnt work. 
Is anyone facing this issue? Can anyone help here?

Management API: https://management.azure.com/providers/Microsoft.Management/getEntities?api-version=2020-05-01

**Comment by josunefon** [comment, marked as answer]
Can you check the **important** section [here](https://github.com/Azure/CCOInsights#cco-azure-infrastructure-dashboard-report-pages) to get resources for another tenant?

**Reply by scarytoon** [reply]
Hi 

Cannot seem to find the information to get resources from another tenant? can you please help?
